src/gurt/telemetry.c

```diff
@@ -1287,11 +1287,11 @@ d_tm_print_stats(FILE *stream, struct d_tm_stats_t *stats, int format)
 		return;
 	}
 
-	fprintf(stream, ", min: %lu, max: %lu, mean: %lf, sample size: %lu",
-		stats->dtm_min, stats->dtm_max, stats->mean,
-		stats->sample_size);
+	fprintf(stream, " [min: %lu, max: %lu, avg: %.0lf",
+		stats->dtm_min, stats->dtm_max, stats->mean);
 	if (stats->sample_size > 2)
-		fprintf(stream, ", std dev: %lf", stats->std_dev);
+		fprintf(stream, ", stddev: %.0lf", stats->std_dev);
+	fprintf(stream, ", samples: %lu]", stats->sample_size);
```

Comment:
I assume that those changes are way the unit tests are failing. @kjacque, would you mind having a look? I checked the unit tests and couldn't find anything obvious. I am ok to revert those changes too if they don't make sense to you.